### PR TITLE
Add Emergency Alert popup

### PR DESCRIPTION
**Pull request by xenophenes**

### CODE REVIEW

File: src/views/Notification/components/NotificationForm/index.jsx [hunk 1]
```diff
@@ -57,7 +56,7 @@ const DistanceSearch = ({ store }) => {
  * @param {function(): void} props.onCancelButtonClick - Callback that gets called when the user
  *     clicks the cancel button
  * @param {function(): void} props.onNotifyButtonClick - Callback that receives the edited
- *     parcel when the user clicks the "Notify" button
+ *     parcel when the user clicks the "Notify" button. Triggers an alert popup on success.
```
Comment (jmezzacappa) [inline src/views/Notification/components/NotificationForm/index.jsx]: Maybe just change this to "Callback that gets called when the user clicks the "notify" button.
The callback no longer receives the edited parcel, and this module doesn't really need to know what happens up the component tree.
(If that functionality changes, it's likely that this documentation will not get updated.)

**Review by jmezzacappa — CHANGES_REQUESTED** [review]

File: src/views/Notification/components/NotificationForm/index.jsx (hunk 1 above)
Comment (xenophenes) [reply src/views/Notification/components/NotificationForm/index.jsx]: Done!

File: src/views/Notification/components/NotificationForm/index.jsx [hunk 2]
```diff
@@ -57,7 +56,7 @@ const DistanceSearch = ({ store }) => {
  * @param {function(): void} props.onCancelButtonClick - Callback that gets called when the user
  *     clicks the cancel button
  * @param {function(): void} props.onNotifyButtonClick - Callback that receives the edited
- *     parcel when the user clicks the "Notify" button
+ *     parcel when the user clicks the "Notify" button.
```
Comment (jmezzacappa) [inline src/views/Notification/components/NotificationForm/index.jsx]: The callback doesn't actually receive a parcel anymore. This documentation is deprecated. Let's just go with "Callback that gets called when the user clicks the notify button"

**Review by jmezzacappa — CHANGES_REQUESTED** [review]

File: src/views/Notification/components/NotificationForm/index.jsx (hunk 2 above)
Comment (xenophenes) [reply src/views/Notification/components/NotificationForm/index.jsx]: Done

**Review by jmezzacappa — APPROVED** [review]

**Pull request merged**